packages/vm/examples/run-blockchain.ts

```diff
@@ -38,31 +30,34 @@ async function setupPreConditions(vm: VM, data: any) {
 
     for (const [key, val] of Object.entries(storage)) {
       const storageKey = setLengthLeft(hexToBytes(key), 32)
-      const storageVal = hexToBytes(val as string)
+      const storageVal = hexToBytes(val as PrefixedHexString)
       await vm.stateManager.putStorage(address, storageKey, storageVal)
     }
 
-    const codeBuf = hexToBytes('0x' + code)
+    const codeBuf = hexToBytes(`0x${code}`)
     await vm.stateManager.putCode(address, codeBuf)
   }
 
   await vm.stateManager.commit()
 }
 
-async function putBlocks(blockchain: Blockchain, common: Common, data: typeof testData) {
+async function putBlocks(blockchain: Blockchain, common: Common, data: typeof blockchainMockData) {
```

Comment:
What is `blockchainMockData`? It is not defined it seems?

Reply:
My bad, forgot to add the `export` before const in the updated test file and that cleaned it up as an unused import. Fixed & pushed